c7n/resources/emr.py

```diff
@@ -75,6 +75,23 @@ def augment(self, resources):
 
 @actions.register('terminate')
 class Terminate(BaseAction):
+    """Action to terminate EMR cluster(s)
+
+    It is recommended to apply a filter to the terminate action to avoid
+    termination of all EMR clusters
+
+    :example:
+
+        .. code-block: yaml
+
+            policies:
+              - name: emr-terminate
+                resource: emr
```

Comment:
Ok, I'll go through each docstring PR I have and once I've ensured all examples are valid & tested, I'll comment on them letting you know.